ddl/column_type_change_test.go

```diff
@@ -2227,3 +2227,14 @@ func (s *testColumnTypeChangeSuite) TestChangeFromBitToStringInvalidUtf8ErrMsg(c
 	errMsg := "[table:1366]Incorrect string value '\\xEC\\xBD' for column 'a'"
 	tk.MustGetErrMsg("alter table t modify column a varchar(31) collate utf8mb4_general_ci;", errMsg)
 }
+
+func (s *testColumnTypeChangeSuite) TestForIssue24621(c *C) {
```

Comment:
Do we need this test anymore? I think it's already coverd by others tests.

Reply:
We can delete this test if we already have one. But would it be nice to say where this case is already covered?

Reply:
A bit like, but not exactly the same. Maybe the best way is to keep it.